source/_integrations/light.mqtt.markdown

```diff
@@ -1044,6 +1049,47 @@ light:
 
 {% endraw %}
 
+### CCT light (brightnes and temperature)
+
+This example comes from configuration of Shelly RGBW Bulb working in White mode. 
+Max_mireds and min_mireds set white color temperature boundaries to 3000K - 6500K. Notice the same limits are applied in `command_on_template`, but in Kelvin units this time. It's due to conversion from Mireds to Kelvin which causes exceeding boundary values accepted by the device.
```

Comment:
- Code-fence the configuration properties so they get highlighted.
- Remove the "white" from the color temperature.
- `Mireds` should not be capitalized.

Reply:
I did check for Kelvin units. It seems it has to be lowercase too (while Kelvin scale is uppercase). Are you ok with that change?
Also added new line to example description:

> The code also ensures bi-directional conversion of brightness scale between 0-100 (required by the device) and 0-255 (required by Home Assistant).

Is it ok?